### DISCUSSION TITLE
Welcome to crowdin-api-client-python Discussions!

### DISCUSSION THREAD

**Discussion by andrii-bodnar** [crowdin/crowdin-api-client-python#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mosesn292-hue** [comment]
Hello please I got the notice on a compensation and I haven't gotten to receive it again
